### PR TITLE
added basic stats to admin page

### PR DESCRIPTION
**Pull request by Gjain234**
screen shot 2018-06-04 at 5 46 00 pm

I basically added a simple getSize function to the various Store classes and then updated their tests as well with simple getSize asserts.

### CODE REVIEW

File: src/main/java/codeu/controller/AdminServlet.java [hunk 1]
```diff
@@ -22,6 +8,8 @@
 
 public class AdminServlet extends HttpServlet {
 
+/** Directs user to admin page.
+ */
```
Comment (beckyqiu) [inline src/main/java/codeu/controller/AdminServlet.java]: one line comment, no need to be two lines here.

/** Directs user to admin page. */

File: src/main/java/codeu/model/store/basic/ConversationStore.java [hunk 2]
```diff
@@ -100,4 +100,11 @@ public Conversation getConversationWithTitle(String title) {
   public void setConversations(List<Conversation> conversations) {
     this.conversations = conversations;
   }
+
+  /**
+   * Returns the current size of the Conversation Store.
+   */
```
Comment (beckyqiu) [inline src/main/java/codeu/model/store/basic/ConversationStore.java]: Same here.

File: src/main/java/codeu/model/store/basic/ConversationStore.java [hunk 3]
```diff
@@ -100,4 +100,11 @@ public Conversation getConversationWithTitle(String title) {
...
+  public int getSize(){
```
Comment (beckyqiu) [inline src/main/java/codeu/model/store/basic/ConversationStore.java]: empty space after ()

File: src/test/java/codeu/model/store/basic/MessageStoreTest.java [hunk 4]
```diff
@@ -61,6 +61,11 @@ public void testGetMessagesInConversation() {
     assertEquals(MESSAGE_TWO, resultMessages.get(1));
   }
 
+  @Test /*before adding message*/
```
Comment (beckyqiu) [inline src/test/java/codeu/model/store/basic/MessageStoreTest.java]: Comment should be added before @Test line.

Comment (beckyqiu) [reply src/test/java/codeu/model/store/basic/MessageStoreTest.java]: Or you can just add this to the function name. For example - testGetSizeBeforeAddingMessage(). That would be better i feel.

File: src/test/java/codeu/model/store/basic/MessageStoreTest.java [hunk 5]
```diff
@@ -75,6 +80,7 @@ public void testAddMessage() {
     messageStore.addMessage(inputMessage);
     Message resultMessage = messageStore.getMessagesInConversation(inputConversationId).get(0);
 
+    Assert.assertEquals(messageStore.getSize(), 4); //check size after adding message
```
Comment (beckyqiu) [inline src/test/java/codeu/model/store/basic/MessageStoreTest.java:79]: I feel you can add the testGetSize function into this test. Assert that size is 3 before sending the msg, and then check it again here to make sure it changed to 4 now.

**Review by beckyqiu — CHANGES_REQUESTED** [review]
Left some comments based on java readability. Please change the format of one line comment to /** blabla. */ and add empty space after () and before {.

Also merging those two tests makes more sense to me. WDYT?

File: src/test/java/codeu/model/store/basic/MessageStoreTest.java (hunk 5 above)
Comment (Gjain234) [reply src/test/java/codeu/model/store/basic/MessageStoreTest.java:79]: I changed the test to fit that model. I agree, I couldn't actually figure out the best way to test the multiple checks of size.

**Review by beckyqiu — APPROVED** [review]
Don't want to block you because of those empty spaces after the function names... :D 
Please also wait for teammates to approve the change to merge code.
Thanks Gauri! Nice job!

**Review by ernesto-rojas — APPROVED** [review]
Hey Gauri, I think this is good!

**Pull request merged**